lib/public/views/Logs/Logs.js

```diff
@@ -224,6 +229,38 @@ export default class LogModel extends Observable {
         }
     }
 
+    /**
+     * Returns the raw current environment filter
+     * @returns {String} the raw current environment filter
+     */
+    getEnvFilterRaw() {
+        return this._environmentFilterRawValue;
+    }
+
+    /**
+     * Returns the current environment filter
+     * @returns {String[]} The current environment filter
+     */
+    getEnvFilter() {
+        return this.environmentFilterValues;
+    }
+
+    /**
+     * Sets the environment filter
+     * @param {String} rawEnvironments The environments to apply to the filter
+     * @returns {undefined}
+     */
+    setEnvFilter(rawEnvironments) {
+        this._environmentFilterRawValue = rawEnvironments;
+        const environmentRegex = /([0-9a-zA-Z]+(?=,?))/g; // Matches comma-separated alphanumeric strings
```

Comment:
Maybe you should also allow for spaces around the env ids?

Reply:
Yes I was specifically thinking between the env id and the comma. This should of course never happen, but sometimes copy-paste may add space and this can be frustrating. 
`envId1 , envId2` should work as `envid1,envId2`